ckanext/datarequests/tests/test_plugin.py

```diff
@@ -40,7 +40,7 @@ def setUp(self):
         self.tk_patch = patch('ckanext.datarequests.plugin.tk')
         self.tk_mock = self.tk_patch.start()
 
-        self.config_patch = patch('ckanext.datarequests.plugin.config')
```

Comment:
does not seem like much of a change

Reply:
The main reason for it was to break the circular dependency between `plugin`, `actions` and `validation`.